src/components/AddDiveForm/AddDive.js

```diff
@@ -1,16 +1,15 @@
 import styled from 'styled-components';
-import { useState } from 'react';
 import { nanoid } from 'nanoid';
 import { IoMdArrowRoundBack as ArrowBack } from 'react-icons/io';
 import Button from '../Button/Button';
+import ScreenReaderOnly from '../ScreenReaderOnly';
 
 export default function AddDive({ onClickBack, onCreate }) {
-  const [diveData, setDiveData] = useState('');
-
   return (
-    <>
+    <Wrapper>
       <ButtonBack name="back" onClick={onClickBack}>
         <ArrowBack />
+        <ScreenReaderOnly>back to divelogs</ScreenReaderOnly>
       </ButtonBack>
       <Headline>log new dive</Headline>
       <FormStyled
```

Comment:
I suggest aria-labelledby="form-title" and id="form-title" at the Headline component. If you do this screenreader is able to translate it if website is translated 🙂 

Aria-label is not translateable (is this an english word?) 🤔  😄